From 5f96d2062de40de85840bd91a729d4424573bfc7 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Sun, 4 Apr 2021 23:45:51 +0200
Subject: [PATCH 2/2] Do not run mruby-file-stat's configure at copmpile time
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Makefile b/Makefile
index 3000ca2..198e3ed 100644
--- a/Makefile
+++ b/Makefile
@@ -3,7 +3,6 @@ all:
 	ruby ./rebuild-fcache.rb
 	cd deps/nanovg/src   && $(CC) $(CFLAGS) nanovg.c -c -fPIC
 	$(AR) rc deps/libnanovg.a deps/nanovg/src/*.o
-	cd deps/mruby-file-stat/src && ../configure
 	cd src/osc-bridge    && make lib
 #	cd mruby             && UI_HOTLOAD=1 MRUBY_CONFIG=../build_config.rb rake
 	cd mruby             && MRUBY_CONFIG=../build_config.rb rake
-- 
2.26.2

