From cf71cc33c44d25a5483ef3ed4ea994f845e0f9b8 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Sat, 25 Jan 2020 17:20:39 +0100
Subject: [PATCH] a-comp.lv2/wscript: sync to master
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Backport

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 libs/plugins/a-comp.lv2/wscript | 12 +++++-------
 1 file changed, 5 insertions(+), 7 deletions(-)

diff --git a/libs/plugins/a-comp.lv2/wscript b/libs/plugins/a-comp.lv2/wscript
index 00099cb564..c739e4bea7 100644
--- a/libs/plugins/a-comp.lv2/wscript
+++ b/libs/plugins/a-comp.lv2/wscript
@@ -15,10 +15,9 @@ def options(opt):
 def configure(conf):
     conf.load('compiler_c')
     autowaf.configure(conf)
-    if Options.options.lv2:
-        autowaf.check_pkg(conf, 'lv2', atleast_version='1.0.0',
-                uselib_store='LV2_1_0_0')
-        autowaf.check_pkg(conf, 'cairo', uselib_store='CAIRO', atleast_version='1.12.0') 
+    autowaf.check_pkg(conf, 'lv2', atleast_version='1.0.0',
+                      uselib_store='LV2_1_0_0')
+    autowaf.check_pkg(conf, 'cairo', uselib_store='CAIRO', atleast_version='1.12.0')
 
 def build(bld):
     bundle = 'a-comp.lv2'
@@ -28,7 +27,6 @@ def build(bld):
     if bld.is_defined('HAVE_LV2'):
         # Build RDF files
         for i in ['manifest.ttl', 'a-comp.ttl', 'a-comp#stereo.ttl', 'presets.ttl']:
-            bld(features     = 'subst',
             obj = bld(features='subst')
             obj.source = i + '.in'
             obj.target = '../../LV2/%s/%s' % (bundle, i)
@@ -40,8 +38,8 @@ def build(bld):
         obj = bld(features     = 'c cshlib',
                   source       = 'a-comp.c',
                   name         = 'a-comp',
-                  cflags       = [ '-fPIC',  bld.env['compiler_flags_dict']['c99'] ],
-                  includes     = [ '../../ardour' ],
+                  cflags       = [ bld.env['compiler_flags_dict']['pic'],  bld.env['compiler_flags_dict']['c99'] ],
+                  includes     = [ '../../ardour', '../shared' ],
                   target       = '../../LV2/%s/a-comp' % bundle,
                   install_path = '${LV2DIR}/%s' % bundle,
                   uselib       = 'CAIRO',
-- 
2.21.0

