From b207c69abfc8fea9b9d126f0366cfd5d84d95806 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Mon, 25 Mar 2019 00:14:49 +0100
Subject: [PATCH] Remove RPATHs
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 scripts/make/tools.mk | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/scripts/make/tools.mk b/scripts/make/tools.mk
index 2adb3c56..0d67881b 100644
--- a/scripts/make/tools.mk
+++ b/scripts/make/tools.mk
@@ -22,7 +22,7 @@ export LD              ?= $(TOOL_LD)
 export MAKE_OPTS        = -s
 export CFLAGS          += $(CC_ARCH) -std=c++98 -fdata-sections -pthread -ffunction-sections -fno-exceptions -fno-asynchronous-unwind-tables -Wall -pipe -fno-rtti $(CC_FLAGS) -DLSP_MAIN_VERSION=\"$(VERSION)\" -DLSP_INSTALL_PREFIX=\"$(PREFIX)\"
 export CXXFLAGS        += $(CC_ARCH) -std=c++98 -fdata-sections -pthread -ffunction-sections -fno-exceptions -fno-asynchronous-unwind-tables -Wall -pipe -fno-rtti $(CC_FLAGS) -DLSP_MAIN_VERSION=\"$(VERSION)\" -DLSP_INSTALL_PREFIX=\"$(PREFIX)\"
-export SO_FLAGS         = $(CC_ARCH) -Wl,-rpath,$(LD_PATH) $(FLAG_RELRO) -Wl,--gc-sections -shared -Llibrary -lc -fPIC
+export SO_FLAGS         = $(CC_ARCH) $(FLAG_RELRO) -Wl,--gc-sections -shared -Llibrary -lc -fPIC
 export MERGE_FLAGS      = $(LD_ARCH) -r
-export EXE_TEST_FLAGS   = $(LDFLAGS) $(CC_ARCH) -Wl,-rpath,$(LD_PATH)
-export EXE_FLAGS        = $(LDFLAGS) $(CC_ARCH) -Wl,-rpath,$(LD_PATH) $(FLAG_RELRO) -Wl,--gc-sections
+export EXE_TEST_FLAGS   = $(LDFLAGS) $(CC_ARCH)
+export EXE_FLAGS        = $(LDFLAGS) $(CC_ARCH) $(FLAG_RELRO) -Wl,--gc-sections
-- 
2.21.0

