From f0e8de8bd963d8766d6ed53d20b1575dad7bc9f5 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Fri, 30 Apr 2021 21:55:54 +0200
Subject: [PATCH] Allow parallel build
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/makefile b/makefile
index 6c026f7..0c094ad 100644
--- a/makefile
+++ b/makefile
@@ -170,4 +170,3 @@ clean:
 
 .PHONY: all install install-strip uninstall clean
 
-.NOTPARALLEL:
-- 
2.30.2

