From 6a3b212c7a945bdd951d2ba1e8293a491ac26a24 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Sun, 18 Jun 2017 14:25:49 +0200
Subject: [PATCH] Keep our build flags
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 src/CMakeLists.txt | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 2900f4b7..fa6f9e10 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -251,15 +251,11 @@ set (BuildOptions_NEON
   CACHE STRING "Cortex_a9 compiler options"
 )
 
-check_cxx_compiler_flag("${BuildOptions_NEON} -Werror" SUPPORT_NEON)
-
 set (BuildOptions_SSE
 	"-msse -msse2 -mfpmath=sse"
   CACHE STRING "SSE compiler options"
 )
 
-check_cxx_compiler_flag("${BuildOptions_SSE} -Werror" SUPPORT_SSE)
-
 set (BuildOptionsBasic
     "-std=c++11 -Wno-unused-parameter -O3 -ffast-math -fomit-frame-pointer"
     CACHE STRING "basic X86 compiler options"
-- 
2.34.1

