From 2354b87b8413d4db220199bf2cea7a84b572e998 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Wed, 13 Jan 2021 09:23:28 +0100
Subject: [PATCH] Work around broken git-version creation
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [workaround]

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 src/Version.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/Version.h b/src/Version.h
index 7d38056c..3184bbd4 100644
--- a/src/Version.h
+++ b/src/Version.h
@@ -19,7 +19,6 @@
 #pragma once
 
 #include "ElementApp.h"
-#include "GitVersion.h"
 
 #ifndef EL_GIT_VERSION
  #define EL_GIT_VERSION ""
-- 
2.26.2

