From 6d35e91b8c5112d5cbe897e919a5ca7fac8979c5 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Sun, 7 Jul 2019 23:41:35 +0200
Subject: [PATCH] build/mixxx.py: Fix build with recent python3
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Pending

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 build/mixxx.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/build/mixxx.py b/build/mixxx.py
index b9501ee81a..e423c88018 100644
--- a/build/mixxx.py
+++ b/build/mixxx.py
@@ -214,15 +214,15 @@ class MixxxBuild(object):
         import subprocess
         process = subprocess.Popen("%s %s" %(self.env['CC'], '--version'), stdout=subprocess.PIPE, shell=True) # nosec
         (stdout, stderr) = process.communicate()
-        self.compiler_is_gcc = 'gcc' in stdout.lower()
-        self.compiler_is_clang = 'clang' in stdout.lower()
+        self.compiler_is_gcc = 'gcc' in stdout.decode("utf-8").lower()
+        self.compiler_is_clang = 'clang' in stdout.decode("utf-8").lower()
 
         # Determine the major compiler version (only GCC)
         if self.compiler_is_gcc:
             self.gcc_major_version = None
             process = subprocess.Popen("%s %s" %(self.env['CC'], '-dumpversion'), stdout=subprocess.PIPE, shell=True) # nosec
             (stdout, stderr) = process.communicate()
-            gcc_version = stdout
+            gcc_version = stdout.decode("utf-8")
             # If match is None we don't know the version.
             if not gcc_version is None:
                 version_split = gcc_version.split('.')
-- 
2.21.0

