From 553cc367bf10401ac2a2849d24c0e93c4157a6d3 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Tue, 20 Aug 2019 00:20:06 +0200
Subject: [PATCH] Do not add host libdir to LDPATH / rpath
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [cross specific]

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 build/depends.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/depends.py b/build/depends.py
index 8604466701..954740fe79 100644
--- a/build/depends.py
+++ b/build/depends.py
@@ -506,7 +506,7 @@ class Qt(Dependence):
         if build.platform_is_osx:
             compiling_on_104 = (
                 os.popen('sw_vers').readlines()[1].find('10.4') >= 0)
-        if not build.platform_is_windows and not (using_104_sdk or compiling_on_104):
+        if False:
             qtdir = build.env['QTDIR']
             framework_path = Qt.find_framework_libdir(qtdir, qt5)
             if os.path.isdir(framework_path):
-- 
2.21.0

