From 766c8c4aa6f6e8743234b93a760a89480bddd7fd Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Mon, 28 May 2018 23:08:07 +0200
Subject: [PATCH] Use native rcgen
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [cross specific]

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 plugingui/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugingui/Makefile.am b/plugingui/Makefile.am
index d102024..8a4a677 100644
--- a/plugingui/Makefile.am
+++ b/plugingui/Makefile.am
@@ -71,7 +71,7 @@ BUILT_SOURCES = \
 	$(NLS_RES)
 
 resource_data.cc: rcgen$(EXEEXT) $(RES) $(NLS_RES)
-	$(rcgen_verbose)./rcgen$(EXEEXT) -s $(top_srcdir)/plugingui/ -s $(top_builddir)/plugingui/ -o $@ $(RES) $(NLS_RES)
+	drumgizmo-rcgen -s $(top_srcdir)/plugingui/ -s $(top_builddir)/plugingui/ -o $@ $(RES) $(NLS_RES)
 
 CLEANFILES = \
 	resource_data.cc \
-- 
2.26.2

