From 5e5395fb24a2fac8e5377aba16e5a47ccfba977f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Thu, 24 Oct 2019 21:47:59 +0200
Subject: [PATCH] Do not create Manifest.ttl with cross ttlgen
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 plugin/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugin/Makefile.am b/plugin/Makefile.am
index 6c0d324..5b36f21 100644
--- a/plugin/Makefile.am
+++ b/plugin/Makefile.am
@@ -66,7 +66,7 @@ UITYPE=CocoaUI
 endif
 
 manifest.ttl : ttlgen drumgizmo.la
-	./ttlgen .libs/drumgizmo.so manifest.ttl $(UITYPE)
+	echo ./ttlgen .libs/drumgizmo.so manifest.ttl $(UITYPE)
 
 
 #######
-- 
2.21.0

