From 21dab1327d0e8121bb4fdc3e0bd50a4f885ef3fc Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Sun, 12 Jun 2022 00:27:53 +0200
Subject: [PATCH] do not pin sse flags - they won't work on all arches
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 meson.build | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/meson.build b/meson.build
index ea26cf1..ccdfcf8 100755
--- a/meson.build
+++ b/meson.build
@@ -22,10 +22,6 @@ current_arch = build_machine.cpu_family()
 # Shared c_args for libraries
 lib_c_args = ['-fvisibility=hidden']
 
-# Add default x86 and x86_64 optimizations
-if current_arch == 'x86' or current_arch == 'x86_64' and current_os != 'darwin'
-    lib_c_args += ['-msse','-msse2','-mfpmath=sse','-ffast-math','-fomit-frame-pointer','-fno-finite-math-only']
-endif
 
 
 # Configure extension for shared object
-- 
2.34.3

