From 7637d0c9c836831eb23af2d72dd8195c65b0aca0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Mon, 9 Dec 2019 20:25:10 +0100
Subject: [PATCH] Fix build for python3-only environments
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Pending

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 waf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/waf b/waf
index 061fb98..3208571 100755
--- a/waf
+++ b/waf
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # encoding: latin-1
 # Thomas Nagy, 2005-2018
 #
-- 
2.21.0

